src/capellambse_context_diagrams/_elkjs.py

```diff
@@ -397,6 +399,16 @@ def _install_required_npm_pkg_versions() -> None:
         if installed.get(pkg_name) != pkg_version:
             _install_npm_package(pkg_name, pkg_version)
 
+proc = subprocess.Popen(
+    ["elk"],
+    executable="/Users/tobiasmessner/PycharmProjects/capellambse-context-diagrams/src/capellambse_context_diagrams/interop/elk",
```

Comment:
I'm not sure yet if we want to have the executable in `capellambse.dirs.user_cache_dir / "elkjs"` or just in the src folder like here. Then we would need `pathlib.Path(__file__).parent / "interop" / "elk"`.